src/core/events.js

```diff
@@ -24,15 +24,6 @@ const subscribeByWatching = (getState, subscribe) => (selector, changeCallback)
 //this function allows to subscribe to a selector and listen for when it changes
 const subscribeToStoreByWatching = subscribeByWatching(getState, subscribe)
 
-const captureCallback = ({ document, documentBack, face }) => {
```

Comment:
This callback used to be fired when the store updates and matches some criteria. I have moved the callback to fire when a certain part of the flow is complete. It has been added to SDK UI in https://github.com/onfido/onfido-sdk-ui/pull/132